apis/python/src/tiledbsoma/__init__.py

```diff
@@ -14,6 +14,12 @@
 )
 from .measurement import Measurement
 from .sparse_nd_array import SparseNDArray
+from .util_tiledb import (
+    stats_disable,
+    stats_dump,
+    stats_enable,
+    stats_reset,
+)
```

Comment:
Would it make sense to export these directly from `libtiledbsoma` and skip the indirection in `util_tiledb.py`?

```suggestion
from .libtiledbsoma import (
    stats_disable,
    stats_dump,
    stats_enable,
    stats_reset,
)
```

Docstrings [can be created directly at the libtiledbsoma level](https://pybind11.readthedocs.io/en/stable/advanced/misc.html#generating-documentation-using-sphinx).

Reply:
super-snazz re-export @thetorpedodog 😎

Reply:
... or nah 😐 

```
Help on built-in function stats_enable in module tiledbsoma.libtiledbsoma:

stats_enable(...) method of builtins.PyCapsule instance
    stats_enable() -> None
```

Doing the re-export leaves us without docstrings, b/c from `pybind11`

Doing it the naïve way I did (wrap & delegate) does give us docstrings (which I forgot to replace from XXX which is my way of saying to-do to myself, sorry for posting this PR prematurely)

Reply:
as I said, [docstrings can be created directly in pybind11](https://pybind11.readthedocs.io/en/stable/advanced/misc.html#generating-documentation-using-sphinx), saving a layer of copypasting.

Reply:
My apologies @thetorpedodog , missed that line -- looking now ...